fission/function.go

```diff
@@ -106,6 +106,19 @@ func fnCreate(c *cli.Context) error {
 			fatal("Need --env argument.")
 		}
 
+		// examine existence of given environment
+		_, err := client.EnvironmentGet(&metav1.ObjectMeta{
+			Namespace: metav1.NamespaceDefault,
+			Name:      envName,
+		})
+		if err != nil {
+			if e, ok := err.(fission.Error); ok && e.Code == fission.ErrorNotFound {
+				fmt.Printf("Environment \"%v\" is not existed, please create it before accessing the function.\n", envName)
```

Comment:
Error message is not correct. `... is not existed, please create...` => `... does not exist. Please create it before creating the function.`

Reply:
`Please create it before creating the function` means that we need to stop the function creation if an environment not existed? I think we can simply show the warning here instead of stopping the creation.

Reply:
Oh right, I thought it was an error. What about making it a bit more explicit `Without the environment the function cannot be executed!`?

Reply:
It is just the word 'accessing' that is bothering me btw. It is too vague. But I am also not sure what word would be best to replace that. ;)

Reply:
How about `Function with non-existing environment cannot be executed. Please create environment "%v" after the function creation.` ?

Reply:
I liked the first sentence as it is very short and direct about what is wrong. The second sentence provides some more explanation. What did you think about the `Without the environment the function cannot be executed!` one?

Reply:
After discussion with @erwinvaneyk on slack, we think it's better to guide a user what should do next when he/she saw the warning message. The decision is 
```
"Environment \"%v\" does not exist. Please create the environment before executing the function. \nFor example: `fission env create --name %v --image <image>`\n"
```